01-introduction.Rmd

```diff
@@ -401,9 +401,9 @@ This includes the **mapsf** package (successor of **cartography**) [@giraud_maps
 
 <!-- spatstat?? -->
 
-In late 2021, the planned retirement of **rgdal**, **rgeos** and **maptools** at the end of 2023 was announced on [the R-sig-Geo mailing list](https://stat.ethz.ch/pipermail/r-sig-geo/2021-September/028760.html) by Roger Bivand.
-This would have a large impact on existing workflows applying these packages, but also will influence the packages that depend on **rgdal**, **rgeos** or **maptools**. 
-Therefore, Bivand's suggestion is to transition to more modern tools, including **sf** and **terra**, as explained in this book's next chapters.
+In late 2021, the planned retirement of **rgdal**, **rgeos** and **maptools** was announced on [the R-sig-Geo mailing list](https://stat.ethz.ch/pipermail/r-sig-geo/2021-September/028760.html) by Roger Bivand.
+This retirement at the end of 2023 have had a large impact on existing workflows applying these packages, but also will influence the packages that depend on **rgdal**, **rgeos** or **maptools**. 
```

Comment:
```suggestion
These packages, and packages that depended on them, were removed from CRAN at the end of 2023.
```

Reply:
There's repetition of `**rgdal**, **rgeos** and **maptools**`.

The typo fix option: 

This retirement at the end of 2023 **has** had ... 

But I think less is more now that they have been removed.